src/pages/Tags/Tags.tsx

```diff
@@ -198,35 +189,21 @@ const Tags: FC = () => {
           />
         </div>
       </Container>
-
       <div className={classes.categoryContainer}>
-        {map(groupedCategoryData, (tags, type: TagTypes) => (
-          <Container key={type} className={classes.category}>
-            <div className={classes.tagCategoryNameContainer}>
-              <span className={classes.categoryName}>
-                {t(`tag.type.${type}`)}
-              </span>
-              <Button
-                appearance={AppearanceTypes.Text}
-                size={SizeTypes.S}
-                icon={EditIcon}
-                className={classes.editIcon}
-                hidden={type === TagTypes.Oskused}
-              >
-                {t('button.change')}
-              </Button>
-            </div>
-
-            <div className={classes.tagCategorySeparator} />
-            <ul>
-              {map(tags, (tag) => (
-                <li className={classes.tagName} key={tag?.id}>
-                  {tag?.name}
-                </li>
-              ))}
-            </ul>
-          </Container>
-        ))}
+        {map(
+          groupedCategoryData,
+          (tagsList: ListDataType[], type: TagTypes) => (
```

Comment:
**[blocker]**

Typing could be improved here. Currently you are rewriting tagsList to be of type ListDataType[] instead of the inferred TagFields[] type. Differences are that name and id are required in ListDataType and not required in TagFields. 

So if "id" and "name" are actually not needed in TagCategory, they shouldn't be required. If they are then we would still need to check for this, since we currently aren't checking for it and we know that in "tagsList" they are actually not required

If they should also be required in TagsList instead, then the change should be done there